modular_zubbers/master_files/code/modules/entombed_quirk/code/entombed_mod.dm

```diff
@@ -48,7 +48,7 @@
 	theme = /datum/mod_theme/entombed
 	applied_cell = /obj/item/stock_parts/power_store/cell/high
 	applied_modules = list(
-		/obj/item/mod/module/storage
+		/obj/item/mod/module/storage/large_capacity  //BUBBER EDIT - was just /storage before
```

Comment:
No need for bubber edit comments on our own files

Reply:
Oh! One sec I'll fix that then!